Tests/IntegrationTests/PSQLIntegrationTests.swift

```diff
@@ -329,4 +329,60 @@ final class IntegrationTests: XCTestCase {
             XCTAssertEqual(obj?.bar, 2)
         }
     }
+    
+#if swift(>=5.5.2)
+    func testBindMaximumParameters() async throws {
+        let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        defer { XCTAssertNoThrow(try eventLoopGroup.syncShutdownGracefully()) }
+        let eventLoop = eventLoopGroup.next()
+        
+        try await withTestConnection(on: eventLoop) { connection in
+            // Mac binds limit is UInt16.max which is 65535 which is 3 * 5 * 17 * 257
+            // Max columns limit is 1664, so we will only make 5 * 257 columns which is less
+            // Then we will insert 3 * 17 rows
+            // In the insertion, there will be a total of 3 * 17 * 5 * 257 == UInt16.max bindings
+            // If the test is successful, it means Postgres supports UInt16.max bindings
+            let columnsCount = 5 * 257
+            let rowsCount = 3 * 17
+            
+            let createQuery = PostgresQuery(
+                unsafeSQL: """
+                CREATE TABLE table1 (
+                \((0..<columnsCount).map({ #""int\#($0)" int NOT NULL"# }).joined(separator: ", "))
+                );
+                """
+            )
+            try await connection.query(createQuery, logger: .psqlTest)
+            
+            var binds = PostgresBindings(capacity: Int(UInt16.max))
+            for _ in (0..<rowsCount) {
+                for num in (0..<columnsCount) {
+                    try binds.append(num, context: .default)
+                }
+            }
+            XCTAssertEqual(binds.count, Int(UInt16.max))
+            
+            let insertionValues = (0..<rowsCount).map { rowIndex in
+                let indices = (0..<columnsCount).map { columnIndex -> String in
+                    "$\(rowIndex * 1_000 + columnIndex + 1)"
+                }
+                return "(\(indices.joined(separator: ", ")))"
+            }.joined(separator: ", ")
```

Comment:
Since we know, we want to write exactly `65535` values, it might be easier to just reach for:
```
let insertionValues = (0..<UInt16.max).map { "$\(Int($0) + 1)" }.joined(separator: ", ")
```

Reply:
Good idea!

Reply:
However, doesn't seem to work since we'll be missing some parentheses.
Currently, and correctly, the values will look like `($1, $2, ...), ($x, $x+1, ...), ...` but with the simplified calculation, we'll be missing the parentheses, which postgres doesn't like.

Reply:
yes makes sense!